pandas/io/json/_json.py

```diff
@@ -501,7 +501,7 @@ def read_json(
         By file-like object, we refer to objects with a ``read()`` method,
         such as a file handle (e.g. via builtin ``open`` function)
         or ``StringIO``.
-    orient : str
+    orient : str, default None
```

Comment:
I think in other docstrings we've been using `orient : str ,optional` for cases like this.

Reply:
I didn't know this. Thanks for the info.
Updating the PR shortly!!